python/dftd4/test_pyscf.py

```diff
@@ -51,12 +52,16 @@ def test_energy_r2scan_d4():
              """
     )
 
+    if with_ecp is True:
+        mol.ecp = "ccecp"
+        mol.build()
```

Comment:
Can we rather have this as an argument for the constructor instead of rebuilding the object?